python/src/equistore/operations/ones_like.py

```diff
@@ -1,6 +1,7 @@
 from typing import List, Union
+import numpy as np
 
-from equistore import TensorBlock, TensorMap
+from equistore import TensorBlock, TensorMap, Labels
```

Comment:
Same issue and fix as above
https://github.com/lab-cosmo/equistore/actions/runs/4417733640/jobs/7745861938#step:7:533

Reply:
same here we remove the Labels import

Reply:
```suggestion
from equistore import TensorBlock, TensorMap
```